### PR TITLE
fix(element): removed WebElement getSize and getLocation for getRect

### PR DESCRIPTION
**Pull request by kolodiy**
As per the selenium4 changelog removed WebElement getSize and getLocation for getRect.

### CODE REVIEW

**Comment by kolodiy** [comment]
I signed it!

**Comment by kolodiy** [comment]
@cnishina, @juliemr can you please review?

File: lib/element.ts [hunk 1]
```diff
@@ -13,9 +13,8 @@ export class WebdriverWebElement {}
 export interface WebdriverWebElement extends WebElement { [key: string]: any; }
 
 let WEB_ELEMENT_FUNCTIONS = [
-  'click', 'sendKeys', 'getTagName', 'getCssValue', 'getAttribute', 'getText', 'getSize',
-  'getLocation', 'isEnabled', 'isSelected', 'submit', 'clear', 'isDisplayed', 'getId',
-  'takeScreenshot'
+  'click', 'sendKeys', 'getTagName', 'getCssValue', 'getAttribute', 'getText', 'getRect',
+  'isEnabled', 'isSelected', 'submit', 'clear', 'isDisplayed', 'getId', 'takeScreenshot'
```
[the remark this answers is not in the document]
Comment (kolodiy) [reply lib/element.ts]: @CrispusDH , I still see `isDisplayed` in [https://github.com/SeleniumHQ/selenium/blob/selenium-4.0.0-alpha-1/javascript/node/selenium-webdriver/lib/webdriver.js](url). Also it is listed in the dependency spec, please see: spec/dependencyTest/seleniumWebdriver_spec.js

[the remark this answers is not in the document]
Comment (cnishina) [reply lib/element.ts]: Yes this is right. I did not clean up this list.

**Comment by kolodiy** [comment]
@cnishina @juliemr can you please merge this PR? We would like to migrate our tests to Protractor6, but this issue currently blocks us.

**Comment by kolodiy** [comment]
@cnishina thanks for the quick reply. We also need a new package version with these changes to be released. 
I saw that the next planned release is 6.0.1-beta.1. Are there any planned dates for this release? 
What about releasing it with these changes for now?  I can help with changelog and PR with version bump.

**Pull request merged**